### PR TITLE
MEN-4107: introduce new chapter: Device-side API

### PR DESCRIPTION
**Pull request by tranchitella**

### CODE REVIEW

File: .passive-blacklist [hunk 1]
```diff
@@ -1,6 +1,7 @@
 06.Artifact-creation
 07.Server-installation
 09.Downloads
-200.API
-201.Troubleshoot
-202.Release-information
+200.Server-side-API
+201.Device-side-API
+202.Troubleshoot
+203.Release-information
```
Comment (lluiscampos) [inline .passive-blacklist:7]: Once we are renaming, could we move these to `300` and `301`? Then we have room to add more "APIs" in the future.

File: 200.Server-side-API/chapter.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: API
+title: Server-side API
 redirect: /../api/development/
```
Comment (lluiscampos) [inline 200.Server-side-API/chapter.md:3]: To the best of my knowledge, thi should work, but have you built it locally and verified?

**Review by lluiscampos — APPROVED** [review]

File: 200.Server-side-API/chapter.md (hunk 2 above)
Comment (tranchitella) [reply 200.Server-side-API/chapter.md:3]: Yes, I tested it

File: .passive-blacklist (hunk 1 above)
Comment (tranchitella) [reply .passive-blacklist]: Done!

**Pull request merged**